hfc.py

```diff
@@ -22,34 +31,47 @@ def __str__(self):
     
 """)
 
+#variable to hold the github username
     gh_name = input("Please enter your github username: ")
 
+#variable req to hold data about the issues created/opened
     req = requests.get("https://api.github.com/search/issues?q=author:{}%20type:pr%20created:%3E2017-09-30%20created:%3C2017-11-01".format(gh_name))
 
-    json_out = req.json()
+    json_out = req.json() #converts values in req variable to json representation
```

Comment:
Please capitalize the first letter on the sentence and add another space right before `#`.